### DISCUSSION TITLE
[HELP] Can't play any steam game

### DISCUSSION THREAD

**Discussion by bronzetuxie** [prasanthrangan/hyprdots#891] [General]
Hi! I am experiencing some problems running any games on Steam. Specifically, I attempted to launch Football Manager 2024 and Counter-Strike 2, but neither game opens. I have tested various Wine/ProtonGe, and even the native Linux executable for Counter-Strike 2 fails to open. I am not sure whether this issue is related to my Arch  configuration. I have installed Arch in minimal mode, following the hyprdots documentation, and I am relatively new to Linux gaming so maybe I forgot to install somethng idk. 

## My specs:
Dell laptop with R5 5600H and RTX 3050
16 gb ram
OS : Arch Linux x86_64 
Kernel : 6.7.4-arch1-1 
GPU Driver : nouveau 

## Video with the issue

https://github.com/prasanthrangan/hyprdots/assets/160269762/4d61a1ef-d24f-47ff-b1a6-0c9b31d68c05

Cs2 even running native doesnt open. Fm24 with protonge opens, but show a black screen for a few seconds and then quit.

Any help will be welcome! Thanks!

**Comment by RookiexCookie** [comment]
man i dont know why thats happening
but can you try adding this in the launch option
`DRI_PRIME=1 %command%`

**Comment by bronzetuxie** [comment]
> man i dont know why thats happening but can you try adding this in the launch option `DRI_PRIME=1 %command%`

Still the same error.
 if I use this launch option: PROTON_USE_WINED3D=1 %command% the game opens. But run on my integrated gpu, I want to run on my nvidia gpu, how can I switch that?

**Comment by RookiexCookie** [comment]
@bronzetuxie do you nvidia drivers properly installed? 
what does `nvidia-smi` outputs on the terminal?

**Comment by bronzetuxie** [comment]
> @bronzetuxie do you nvidia drivers properly installed? what does `nvidia-smi` outputs on the terminal?

I think I don't, because nvidia-smi send me a unknown command on terminal, how can I install? On neofetch says nouveau, it's the open source nvidia driver

**Comment by hchasens** [comment]
Please refer to the following:
https://wiki.archlinux.org/title/NVIDIA
https://wiki.archlinux.org/title/steam
https://wiki.archlinux.org/title/gaming

Check the wiki pages above. Nouveau is an open-source driver for Nvidia. It's strange the drivers didn't automatically install during Hyprdot's install script. Alternatively, try to reinstall/update Hyprdots and check to make sure the drivers installed. If it still fails to work, try to manually install them.

It should be noted that this is probably not an issue with Hyprdots but with something else in your Arch install. It sounds like it could be a driver issue. Unless Hyprdot failed to install/configure your drivers and your drivers should have been covered by the install; I suggest this be marked as closed as it's out of scope for this project. Should it be closed there are plenty of other resources to help you with your problem.

Can you send us your output to `lsmod` and `lspci -k`.

**Reply by bronzetuxie** [reply]
`lspci -k | grep -A 2 -E "(VGA|3D)"`
01:00.0 VGA compatible controller: NVIDIA Corporation GA107M [GeForce RTX 3050 Mobile] (rev a1)
	Subsystem: Dell GA107M [GeForce RTX 3050 Mobile]
	Kernel driver in use: nouveau
--
06:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Cezanne [Radeon Vega Series / Radeon Vega Mobile Series] (rev c6)
	Subsystem: Dell Cezanne [Radeon Vega Series / Radeon Vega Mobile Series]
	Kernel driver in use: amdgpu

**Comment by kRHYME7** [comment]
@bronzetuxie  👋 I'll be moving this to discussions as this is totally unrelated to config files.

**Comment by bronzetuxie** [comment]
I have installed nvidia package and now completely broke my system, can't even boot. On grub says "you need to load kernel first"

**Comment by hchasens** [comment]
I'm sorry that happened to you. Something may have gone wrong during Section 1 step 5 of the Nvidia entry on the Arch Linux wiki. 

Did you only have the one kernel installed or did you have a backup LTS kernel? If you have your original ArchOS USB installer you should be able to repair it even if you lost your only kernel. 

Since you're running a 3050 you're going to want to download the `nvidia-open` package.

Rather than trying to 

```Remove kms from the HOOKS array in /etc/mkinitcpio.conf and [regenerate the initramfs](https://wiki.archlinux.org/title/Regenerate_the_initramfs). This will prevent the initramfs from containing the nouveau module making sure the kernel cannot load it during early boot.```

It might be worth just skipping that step and rebooting. The nvidia driver packages contain a package called `nvidia-util` as a dep which will automatically blacklist `nouveau`.

Once you've rebooted run `lspcie -k` and see which driver is running. If it boots into `nouveau` then try to remove the hook. I'm sorry you're having so many issues with this.
